### PR TITLE
#1-4Task. aliaksei.paleshchuk.ais

### PR DESCRIPTION
**Pull request by alexey1986**

### CODE REVIEW

**Comment by alexey1986** [comment]
https://travis-ci.org/github/alexey1986/js-assignments

File: task/02-numbers-tasks.js [hunk 1]
```diff
@@ -202,7 +207,18 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    if (n===1) {
+        return false;
+    } else if (n === 2) {
+        return true;
+    } else {
+        for(var x = 2; x < n; x++) {
```
Comment (FedorovD) [inline task/02-numbers-tasks.js:215]: `var` is deprecated. It has no sense to use for browsers code even currently.

File: task/03-date-tasks.js [hunk 2]
```diff
@@ -75,8 +75,19 @@ function isLeapYear(date) {
...
-function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+function timeSpanToString(startDate, endDate) {   
+   var timeSpan = endDate - startDate;
+   var sec_num = timeSpan / 1000;
+   var hh   = Math.floor(sec_num / 3600);
+   var mm = Math.floor((sec_num - (hh * 3600)) / 60);
+   var ss = sec_num - (hh * 3600) - (mm * 60);
+   var sss = parseFloat(ss).toFixed(3);
+
```
Comment (FedorovD) [inline task/03-date-tasks.js:85]: `var` is deprecated. It has no sense to use for browsers code even currently.

File: task/03-date-tasks.js [hunk 3]
```diff
@@ -94,7 +105,19 @@ function timeSpanToString(startDate, endDate) {
...
+   var hours = date.getUTCHours();
+   var minutes = date.getUTCMinutes();
+
+   hours = hours % 12;
+
+   var hourMinPart = 0.5 * minutes;
+   var hourHourPart = 30 * hours;
+   var minAngle = 6 * minutes;
+   var totalAngle = Math.abs(hourMinPart + hourHourPart - minAngle);   
+
```
Comment (FedorovD) [inline task/03-date-tasks.js:117]: `var` is deprecated. It has no sense to use for browsers code even currently.

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**